### ISSUE TITLE
Low accuracy when training withou background

### ISSUE THREAD

**Issue by IgorMunizS** [michalfaber/keras_Realtime_Multi-Person_Pose_Estimation#113]
Hi,
Could you please tell me why when I train without heatmap background, acc do not increase?
The loss decrease normally, but acc never increase. And predicted heatmaps are gray images.

and why do you subtract from 1.0 here?
heatmap[:, :, -1] = np.clip(1.0 - np.amax(heatmap, axis=2), 0.0, 1.0)

the background turns white...

Thanks!

**Comment by rosivagyok** [comment]
Hi @IgorMunizS .

> Could you please tell me why when I train without heatmap background, acc do not increase?

In case of pose estimation accuracy is not a valid KPI, you should evaluate your model based on validation loss or measure mAP after some epochs. That should give feedback based on where your model's "accuracy" stands. 

> and why do you subtract from 1.0 here? heatmap[:, :, -1] = np.clip(1.0 - np.amax(heatmap, axis=2), 0.0, 1.0)

Think of the background channel as your true negative samples for the network, that allows the cpm to distinguish keypoints from everything else. Therefore, background becomes everything else than your generated ground truth keypoint gaussians, meaning the highest activations (1.0) will be background pixels in this case.
